clients/go/pkg/zbc/client.go

```diff
@@ -255,3 +258,12 @@ func configureKeepAlive(config *ClientConfig) error {
 
 	return nil
 }
+
+func getVersion() string {
+	zbVersion := "development"
+	if readVersion, err := embedded.Asset("VERSION"); err == nil {
+		zbVersion = strings.TrimSpace(string(readVersion))
+	}
```

Comment:
Should we log errors? In Java I'd say let's log it at debug level, but is that a thing with the Go logger? :sweat_smile: 

On the other hand, I guess you could tell from the user agent if something goes wrong...still :thinking:

Reply:
It's not, no. I'm not sure what actionable thing we can log here. What do you have in mind? There's nothing for the user to do here, except maybe to contact the team or update. And also, it's not crucial since we have a fallback behavior

Reply:
I guess I'm worried in case something goes wrong in the release process - we regenerate this for the release but don't really test it, right? We test before with a different file, correct? But I guess the chances are quite low...and worse comes to worst, we could add it as a post-release test (e.g. like the get started stuff).

Reply:
True, I can test it in the next release to make sure everything is ok but, even if the version is corrupted somehow, it's also not a very severe bug. It won't break anything